docs/_guides/cron-event-driver.md

```diff
@@ -0,0 +1,92 @@
+# Event Driver: cron
+The `cron` event driver fires events on a schedule determined by the cron expression passed to the event driver when created. This allows Dispatch to trigger functions in response to events generated on a scheduled basis.
+
+## Getting Started
+This guide assumes that you have a working Dispatch instance and access to the Dispatch CLI configured to work with your Dispatch instance. If you need helping creating a Dispatch instance or configuring your Dispatch CLI see the quickstart guide [here](../_front/quickstart.md)
+
+### Create a Function
+For this walkthrough we will use the hello.py example available [here](../../examples/python3). To begin we will seed Dispatch with the images we need for this function.
+
+```bash
```

Comment:
If they followed the quickstart, they already have the seed images.  You may want to state that they only need to run create-seed if the images haven't already been created.